ignite/pkg/protoanalysis/protoutil/creator_test.go

```diff
@@ -0,0 +1,577 @@
+package protoutil
```

Comment:
Can we move these to a separate `protoutil_test` package?

Reply:
I believe so, most other packages go with white box testing so I followed suit. I do *believe* I only use exported names here so it should be doable.

Reply:
well, `isString` is tested but that could also be tested indirectly by adding further checks for creating literals.